how-to/workspace-platform-starter/client/src/framework/launch.ts

```diff
@@ -385,33 +385,80 @@ async function launchView(viewApp: PlatformApp): Promise<PlatformAppIdentifier |
 			const platform = getCurrentSync();
 
 			if (!isEmpty(viewApp.launchPreference?.bounds) || !isEmpty(viewApp?.launchPreference?.options?.view)) {
-				const preferenceWindow = await platform.createWindow({
-					workspacePlatform: !isEmpty(viewApp.launchPreference?.options?.view?.hostUrl)
-						? {
-								windowType: "platform"
-						  }
+				let workspacePlatform:
+					| {
+							disableMultiplePages?: boolean;
+							windowType?: WindowType;
+							favicon?: string | undefined;
+							title?: string;
+							pages?: [
+								{
+									title?: string;
+									iconUrl?: string;
+									layout?: unknown;
+								}
+							];
+							toolbarOptions?: { buttons: [] } | undefined;
+					  }
+					| undefined = {};
```

Comment:
To keep the types stricter this object would be better types as 
```
let workspacePlatform: Partial<BrowserWorkspacePlatformWindowOptions> |
				{ windowType: WindowType.Platform } |
					undefined = {};
```

Reply:
done